crates/storage-proto/src/lib.rs

```diff
@@ -1053,57 +1070,61 @@ pub mod storage {
 
             impl From<restate_types::journal::enriched::EnrichedEntryHeader> for EnrichedEntryHeader {
                 fn from(value: restate_types::journal::enriched::EnrichedEntryHeader) -> Self {
+                    // No need to write down the requires_ack field for any of the entries
```

Comment:
Yes, by definition of requires_ack, if it reached the journal storage then either there is one in-flight stream that was already acked, or there are no in-flight streams and the entry don't need any ack because it's in the replayed journal.